docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

```diff
@@ -115,7 +99,7 @@ This will install the `npm` modules required to properly leverage the Stencil ev
 <!-- theme: info -->
 
 ### Themes Prior to 1.10.0
-> For instructions on installing modules via `jspm` on pre 1.10.0 themes, see: [Installing Legacy Theme Modules](https://developer.bigcommerce.com/legacy/stencil-themes/installing-legacy-theme-modules).
+> For instructions on installing modules via `jspm` on pre 1.10.0 themes, see [Installing Legacy Theme Modules](https://developer.bigcommerce.com/legacy/stencil-themes/installing-legacy-theme-modules).

```

Comment:
@bookernath should we completely deprecate this section since it will be no longer valid for new theme development. Also the page it is linking to ?

Reply:
I agree. It's been a few years, let's remove references to jspm.